docs/installation/setup.rst

```diff
@@ -11,131 +11,137 @@ Prerequisites
 .. _SaltStack: https://www.saltstack.com
 .. _Puppet: https://puppet.com
 
-MetalK8s_ clusters require machines running CentOS_ / RHEL_ 7.6 or higher as
+MetalK8s_ clusters require machines running CentOS_//RHEL_ 7.6 or higher as
```

Comment:
Why double the slash? Doesn't make sense to me.

Reply:
That was an attempt to escape the slash. Sphinx behaves oddly when links get stacked next to slashes. Nonetheless, this does not render correctly. Fixing.